cws-ui/src/main/webapp/js/adaptation-workers-modal.js

```diff
@@ -0,0 +1,6 @@
+// override with js code file in your adaptation project for example like this
```

Comment:
@zonagit Can you add a sentence or two to the comments here, that describe what this is, and why a project might want to create adaptation content here?  Like what it will result in, and perhaps an example from M20.

Reply:
@galenatjpl  Added it.